extensions/amp-apester-media/0.1/amp-apester-media.js

```diff
@@ -291,6 +292,29 @@ class AmpApesterMedia extends AMP.BaseElement {
     return overflow;
   }
 
+  /**
+   * @param {JsonObject} publisher
+   * @return {Element}
+   */
+  constructAmpPixel_(publisher) {
+    if (publisher && publisher['trackingPixel']) {
```

Comment:
@IdanLanger One question I have regarding the privacy review is that, where does this `publisher['trackingPixel']` come from? Does it mean publisher needs to opt-in to tracking on your server side to enable this feature? Thanks.

Reply:
@zhouyx - Yes. That's exactly where it comes from.

Reply:
Thank you @IdanLanger for the clarification. 

In this case I don't feel strong about a privacy review. But please allow us sometime to get the response from the PWG. Thank you

Reply:
Hi @zhouyx,
Do have any updates regarding the PR?
Thanks.

Reply:
sorry for the wait. haven't heard back from Privacy Working Group yet. 

The original concern was for users who adopted Apester Media would suddenly start to send user data to Apster Media without a consent or notification. But I agree with Yuxuan, since you've already handled the data collection opt-in, the version bump rule should not apply here.

I think we can move forward with the PR. Could you please switch to `CustomEventReporter` ? /cc @zhouyx

Reply:
Hi @lannka.
We committed the changes requested and used `CustomEventReporter`.

However after commuting the changes, we noticed it sends a `POST` request instead of a `GET` request as we intended.

What would you suggest in order for us to trigger a `GET` request?
**(please don't merge)**

Reply:
@lannka - committed changes using `CustomEventReporter`.

Reply:
@lannka ping 😄